README.md

```diff
@@ -47,6 +47,41 @@ $ npm run generate
 - Suggestions Page (Suggestions.vue)
   - ~\screens\SuggestionModal.vue
 
+### Folders
+
+- Assets
+  - CSS/SCSS Files
+  - Fonts
+    - One Folder per font
+      - all font files
+      - a font style sheet
+  - Images, the ones that are unique to each page
+  - SVG files
+  - Utils(not to be confused with ~\utils), idk either its a big mystery to me.
+
+- Components
+  - Elements within pages, like notes, groups, questions, etc.
+- Layouts
+  - Boilerplates
+- Middleware
+  - a very special plugin that won't go into the plugins folder
```

Comment:
be a bit more specific?

Reply:
Middleware is for routing and handling authentication from the server side